tests/e2e/vsan_stretched_cluster_utils.go

```diff
@@ -361,3 +374,141 @@ func createPodsInParallel(client clientset.Interface, namespace string, pvclaims
 		lock.Unlock()
 	}
 }
+
+// updatePvcLabelsInParallel updates the labels of pvc in a namespace in parallel
+func updatePvcLabelsInParallel(ctx context.Context, client clientset.Interface, namespace string,
+	labels map[string]string, pvclaims []*v1.PersistentVolumeClaim, wg *sync.WaitGroup) {
+	defer wg.Done()
+	for _, pvc := range pvclaims {
+		framework.Logf(fmt.Sprintf("Updating labels %+v for pvc %s in namespace %s",
+			labels, pvc.Name, namespace))
+		pvc.Labels = labels
+		_, err := client.CoreV1().PersistentVolumeClaims(namespace).Update(ctx, pvc, metav1.UpdateOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	}
+}
+
+// updatePvLabelsInParallel updates the labels of pv in parallel
+func updatePvLabelsInParallel(ctx context.Context, client clientset.Interface, namespace string,
+	labels map[string]string, persistentVolumes []*v1.PersistentVolume, wg *sync.WaitGroup) {
+	defer wg.Done()
+	for _, pv := range persistentVolumes {
+		framework.Logf(fmt.Sprintf("Updating labels %+v for pv %s in namespace %s",
+			labels, pv.Name, namespace))
+		pv.Labels = labels
+		_, err := client.CoreV1().PersistentVolumes().Update(ctx, pv, metav1.UpdateOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	}
+}
+
+// getMasterIpOnSite returns IP address of master node on a particular site. This function has to be
+// only used while testing distributed - SEB testbed deployment model.
```

Comment:
remove SEB here
just mention that this should be used in deployments where k8s masters are spread across sites